packages/db/src/migrations/0020_add_article.ts

```diff
@@ -0,0 +1,29 @@
+import { type Kysely, sql } from "kysely"
+import { createTableWithDefaults } from "../utils"
+
+export async function up(db: Kysely<any>) {
+  await createTableWithDefaults("articles", { id: true, createdAt: true, updatedAt: true }, db.schema)
+    .addColumn("title", "text", (col) => col.notNull())
+    .addColumn("author", "text", (col) => col.notNull())
+    .addColumn("photographer", "text", (col) => col.notNull())
```

Comment:
Won't there be situations where there is no obvious photographer for an image? Such as ai generated images etc. Do we want to enforce having a photographer on the db level?

Reply:
I think in these cases it should be attributed to the image generator, what do you think?

Reply:
I think it's probably nice to have a required image alt text field here too

Reply:
Yeah it actually seems nice to always have a photographer and alt text